synapse/types.py

```diff
@@ -693,6 +670,7 @@ class StreamToken:
     push_rules_key: int
     to_device_key: int
     device_list_key: int
+    # Note that the groups key is no longer used and may have bogus values.
     groups_key: int
```

Comment:
I'm unsure if there is something better to do here? I did debate setting the default for `groups_key` to 0 so that we don't need to provide it when creating a `StreamToken`?